src/MessageHandler.py

```diff
@@ -0,0 +1,112 @@
+import json
+import urllib2
+from src.MessageRouter import MessageRouter
+
+
+def new_message_handler(config):
+    def _new_command_handlers():
+        return {
+            'casts': _new_casts_handler(),
+            'streams': new_get_streams_response(fetch_url_contents),
+            'nyan': lambda message: ['~=[,,_,,]:3']
```

Comment:
Since this is python, everything is dynamic anyway (Even what you would normally consider static)